plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -109,20 +109,27 @@ const generatedColumns: TableColumn[] = [
     title: 'Build',
     field: 'buildName',
     highlight: true,
-    render: (row: Partial<CITableBuildInfo>) => (
-      <Link
-        component={RouterLink}
-        to={generatePath(buildRouteRef.path, {
-          branch: row.source?.branchName!,
-          buildNumber: row.buildNumber?.toString()!,
-        })}
-      >
-        {row.buildName}
-      </Link>
-    ),
+    render: (row: Partial<CITableBuildInfo>) => {
+      if (!row.source?.branchName || !row.buildNumber) {
+        return <>{row.buildName}</>;
+      }
+
+      return (
+        <Link
+          component={RouterLink}
+          to={generatePath(buildRouteRef.path, {
+            branch: row.source?.branchName,
+            buildNumber: row.buildNumber?.toString(),
```

Comment:
nit: now you don't need these two question marks :)

Reply:
You are right, I will fix that

Reply:
Done :)